scripts/systemd/nutshutdown.in

```diff
@@ -1,2 +1,14 @@
 #!/bin/sh
-@SBINDIR@/upsmon -K >/dev/null 2>&1 && @SBINDIR@/upsdrvctl shutdown
+
+if @SBINDIR@/upsmon -K >/dev/null 2>&1; then
+  wait_delay=`/bin/sed -ne 's#^ *POWEROFF_WAIT= *\(.*\)$#\1#p' /etc/nut/nut.conf`
```

Comment:
As an `.in` template this should not refer to specific pathnames (`etc` => `@sysconfdir@`)... 

And I'm not fully sure about `/bin/sed` ;)